sympy/logic/boolalg.py

```diff
@@ -2186,15 +2215,17 @@ def _get_odd_parity_terms(n):
     Returns a list of lists, with all possible combinations of n zeros and ones
     with an odd number of ones.
     """
-    return [e for e in [ibin(i, n) for i in range(2**n)] if sum(e) % 2 == 1]
+    return [[1 if (mask >> i) & 1 else 0 for i in range(n)]
+            for mask in range(1 << n) if mask.bit_count() % 2 == 1]
```

Comment:
The `int.bit_count` method does not exist in Python 3.9 so this should be changed to use something else.